app/controllers/amr_heatmap_controller.rb

```diff
@@ -1,3 +1,6 @@
+URL_CARD_ARO = "https://card.mcmaster.ca/aro/".freeze
+URL_CARD_INDEX = "https://raw.githubusercontent.com/arpcard/aro/master/aro.owl".freeze
```

Comment:
one more comment while you're making changes :O I noticed this file aro.owl is actually fairly large, so that means a lot of repeated calls to an external service. If it's static and not expected to change often, we should just vendor it and store a version in, say, idseq-database on S3.

Reply:
Vendorized!